src/bio2bel/sources/biogrid.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+
+"""This script downloads and parses BioGRID data and maps the interaction types to BEL."""
+
+import logging
+from typing import Iterable, List
+
+import numpy as np
+import pandas as pd
+from protmapper.uniprot_client import get_mnemonic
+from tqdm import tqdm
+
+import pybel.dsl
+from pybel import BELGraph
+
+
+SEP = '\t'
+BIOGRID = 'biogrid'
+SOURCE = 'source'
+TARGET = 'target'
+ALT_SOURCE_ID = 'alt_source_id'
+ALT_TARGET_ID = 'alt_target_id'
+RELATION = 'relation'
+PUBMED_ID = 'pubmed_id'
+UNIPROT = 'uniprot'
+EVIDENCE = 'From BioGRID'
+MODULE_NAME = 'biogrid'
+
+VERSION = '3.5.183'
+BASE_URL = 'https://downloads.thebiogrid.org/Download/BioGRID/Release-Archive'
+URL = f'{BASE_URL}/BIOGRID-{VERSION}/BIOGRID-ALL-{VERSION}.mitab.zip'
+
+#: Relationship types in BioGRID that map to BEL relation 'increases'
+BIOGRID_INCREASES_ACTIONS = {
+    'synthetic genetic interaction defined by inequality',
+    'additive genetic interaction defined by inequality',
+}
+
+#: Relationship types in BioGRID that map to BEL relation 'decreases'
+BIOGRID_DECREASES_ACTIONS = {
+    'suppressive genetic interaction defined by inequality',
+}
+
+#: Relationship types in BioGRID that map to BEL relation 'association'
+BIOGRID_ASSOCIATION_ACTIONS = {
+    'direct interaction',
+    'physical association',
+    'colocalization',
+    'association',
+}
+
+BIOGRID_COLUMN_MAPPER = {
```

Comment:
why remap the column names from original arbitrary names to new arbitrary names? This just obfuscates the code. Please remove and just use the original headers.

Reply:
I did it to have consistent files that can be used easily. So IntAct and BioGRID both have the same headers, which enables some tasks to be easier and not to require more code. So in this case I would suggest to actually leave them because they have a purpose.